### PR TITLE
Add org.kde.PlatformTheme.QtSNI

### PR DESCRIPTION
**Pull request by ilya-fedin**

### CODE REVIEW

File: org.kde.PlatformTheme.QtSNI.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p ${FLATPAK_DEST}/share/appdata",
+                "cp org.kde.PlatformTheme.QtSNI.appdata.xml ${FLATPAK_DEST}/share/appdata",
+                "appstream-compose --basename=org.kde.PlatformTheme.QtSNI --prefix=${FLATPAK_DEST} --origin=flatpak org.kde.PlatformTheme.QtSNI"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "org.kde.PlatformTheme.QtSNI.appdata.xml"
```
Comment (LeandroStanger) [inline org.kde.PlatformTheme.QtSNI.json:37]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Comment (ilya-fedin) [reply org.kde.PlatformTheme.QtSNI.json]: Fixed

**Comment by bilelmoussaoui** [comment]
bot, build org.kde.PlatformTheme.QtSNI

**Comment by nedrichards** [comment]
The packaging looks fine - but I'm not an expert on the Qt integration and similar @grulja would it be OK if you could take a quick look?

**Comment by grulja** [comment]
Looks good to me. Will this work together with QGnomePlatform and the internal xdgdesktopportal platform theme?

**Comment by ilya-fedin** [comment]
> Will this work together with QGnomePlatform and the internal xdgdesktopportal platform theme?

Ye, it will

**Pull request closed without merging**